org.lflang/src/org/lflang/validation/LFValidator.java

```diff
@@ -55,6 +57,7 @@
 import org.eclipse.xtext.validation.Check;
 import org.eclipse.xtext.validation.CheckType;
 import org.eclipse.xtext.validation.ValidationMessageAcceptor;
+import org.jetbrains.annotations.NotNull;
```

Comment:
I don't think we want to introduce this dependency. It will break things for Eclipse users.

Reply:
I build the eclipse product, and it looks like Epoch isn't broken by this dependency.

Reply:
:point_up:

Reply:
But it also seems like this can be safely removed.

Reply:
How did you build Epoch? In Eclipse?

Reply:
The point is not that it would break Epoch, it would prevent developers from building Epoch _in Eclipse_. This is because Eclipse doesn't use Maven or Gradle but it's own build system that is severely deprived from otherwise generally accessible dependencies like this one. We've run into this issue several times before...

Reply:
Okay, got it...